internal/service/project/resource_project_test.go

```diff
@@ -144,7 +144,7 @@ func TestGetProjectPropsFromAPI(t *testing.T) {
 			readGetManagedSlowMsResponse := http.Response{StatusCode: 200, Body: io.NopCloser(strings.NewReader(managedSlowMsJSON))}
 			perfMock.EXPECT().GetManagedSlowMsExecute(mock.Anything).Return(&readGetManagedSlowMsResponse, nil).Maybe()
 
-			_, err := project.GetProjectPropsFromAPI(context.Background(), projectsMock, teamsMock, perfMock, dummyProjectID)
+			_, err := project.GetProjectPropsFromAPI(context.Background(), projectsMock, teamsMock, perfMock, dummyProjectID, nil)
```

Comment:
why do we need to pass `nil` in the test?

Reply:
we need to pass the param for getting the possible warnings, in this case we don't care if we get some